target/linux/ipq40xx/image/generic.mk

```diff
@@ -549,6 +549,25 @@ define Device/ezviz_cs-w3-wd1200g-eup
 endef
 TARGET_DEVICES += ezviz_cs-w3-wd1200g-eup
 
+define Device/glinet_gl-a1300
+        $(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := GL.iNet
+	DEVICE_MODEL := GL-A1300
+	SOC := qcom-ipq4018
+	DEVICE_DTS_CONFIG := config@ap.dk01.1-c2
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	IMAGE_SIZE := 131072k
+	KERNEL_INSTALL := 1
+	KERNEL_IN_UBI := 1
+	DEVICE_PACKAGE := ipq-wifi-glinet_gl-a1300
+	IMAGES := nand-factory.img sysupgrade.tar
+	IMAGE/nand-factory.img := append-ubi
+	IMAGE/sysupgrade.tar := sysupgrade-tar | append-gl-metadata
```

Comment:
Here should be `append-metadata`. However, `Device/UbiFit` has declared the image file, and these repeated lines (`IMAGE...`) can be deleted.